lib/web_ui/lib/src/engine/view_embedder/flutter_view_manager.dart

```diff
@@ -15,13 +15,18 @@ class FlutterViewManager {
   // A map of (optional) JsFlutterViewOptions, indexed by their viewId.
   final Map<int, JsFlutterViewOptions> _jsViewOptions =
       <int, JsFlutterViewOptions>{};
+  // A map of Stream subscribers for a viewId.
+  final Map<int, List<StreamSubscription<Object?>>> _subscriptions =
+      <int, List<StreamSubscription<Object?>>>{};
   // The controller of the [onViewsChanged] stream.
   final StreamController<void> _onViewsChangedController =
       StreamController<void>.broadcast();
 
   /// A stream of `void` events that will fire when a view is registered/unregistered.
   Stream<void> get onViewsChanged => _onViewsChangedController.stream;
 
+  /// Should we create a different onViewResized stream?
```

Comment:
Should we propagate resize events from this class at all, or should these subscriptions live elsewhere?

Reply:
If we always re-render when there is a resize, then I don't think we need a stream for resize events. I would rather have resize trigger a re-render at a higher level in the engine/framework than at the Renderer level since it doesn't seem right to me for the Renderer to decide when to re-render. I think the Renderer should only perform Render requests and never initiate them.